src/main/java/com/openlattice/datastore/edm/controllers/EdmController.java

```diff
@@ -421,6 +415,31 @@ public Void createEmptySchema( @PathVariable( NAMESPACE ) String namespace, @Pat
                 .map( modelService::getEntitySet )::iterator;
     }
 
+    @Override
+    @RequestMapping(
+            path = SUMMARY_PATH,
+            method = RequestMethod.GET )
+    public Map<UUID, Iterable<PropertyUsageSummary>> getAllPropertyUsageSummaries() {
+        Set<UUID> propertyTypeIds = modelService.getAllPropertyTypeIds();
+        //removes AUDIT types, which don't get populated locally
+        propertyTypeIds.remove(UUID.fromString( "f1a0bda3-406a-42d4-a24b-79e1042a1535" ));
```

Comment:
It's a bit safer/more generic to get the audit property types by loading them from their FQNs (`AuditEntitySetUtils.DETAILS_PT_FQN` and `AuditEntitySetUtils.TYPE_PT_FQN`), but it sounds like we're probably deleting those anyway